tools/api_proto_breaking_change_detector/buf.yaml

```diff
@@ -0,0 +1,23 @@
+version: v1beta1
+# commenting these out for now until I figure out how to automate buf.lock
+#deps:
+  # TODO: should these point to specific commit hashes?
+  # 
+
+#    - buf.build/beta/udpa
+#    - buf.build/beta/googleapis
+#    - buf.build/beta/opencensus
+#    - buf.build/beta/prometheus
+#    - buf.build/beta/opentelemetry
+breaking:
+  use:
```

Comment:
This is the list of breaking change rules we're currently using

Reply:
Generally we'd recommend using one of the categories to match what you want https://docs.buf.build/breaking-rules instead of choosing individual rules. Note that for a given configuration version, the rules within each category are stable, i.e. if you choose `PACKAGE` or `WIRE_JSON` (which look to be roughly what you're going for), we won't add new rules and break your code as long as `version` is `v1beta1`.